tsconfig.json

```
@@ -1,15 +1,16 @@
 {
   "compilerOptions": {
     "incremental": true,
-    "target": "es6",
-    "module": "commonjs",
+    "target": "esnext",
+    "module": "NodeNext",
     "resolveJsonModule": true,
     "sourceMap": true,
     "outDir": "dist",
     "strict": true,
     "noImplicitAny": true,
-    "moduleResolution": "node",
+    "moduleResolution": "NodeNext",
     "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
```

something fun I rediscovered yesterday that may prove useful later on.
it ensures that casing is correct in imports
```suggestion
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
```

---

thanks, configuration added, and code tested without issues.